benchmark/utils/DriverUtils.swift

```diff
@@ -227,27 +231,53 @@ func stopTrackingObjects(_: UnsafeMutableRawPointer) -> Int
 #endif
 
 class SampleRunner {
```

Comment:
Why not refactor the sample runner into a linux/os x version? It would be better than doing the #if all over the place from a code quality perspective.